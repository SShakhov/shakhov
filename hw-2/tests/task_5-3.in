1 1 65335 12 3 1.1 -1.1 4.6789 5.893 -66.66
