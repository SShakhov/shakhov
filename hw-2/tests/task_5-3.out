65352	-56.088100
